pinot-core/src/test/java/org/apache/pinot/core/realtime/impl/fakestream/FakeStreamLevelConsumer.java

```diff
@@ -26,6 +26,7 @@
  * Test implementation of {@link StreamLevelConsumer}
  * This is currently a no-op
  */
+@Deprecated(since = "Pinot no longer support high level consumer model since v0.12.*")
```

Comment:
Should we just clean them up?

Reply:
yes. after removing the integ test, this doesn't make sense. will remove it.